Pod/Classes/CDTQUnindexedMatcher.m

```diff
@@ -178,24 +178,32 @@ - (BOOL)executeSelectorTree:(CDTQQueryNode *)node onRevision:(CDTDocumentRevisio
         BOOL invertResult = [operator isEqualToString:NOT];
         if (invertResult) {
             operatorExpression = operatorExpression[NOT];
-            operator= operatorExpression.allKeys[0];
+            operator = operatorExpression.allKeys[0];
         }
 
         NSObject *expected = operatorExpression[operator];
         NSObject *actual = [CDTQValueExtractor extractValueForFieldName:fieldName fromRevision:rev];
-
+        // Since $in is the same as a series of $eq comparisons -
+        // Treat them the same by:
+        // - Ensuring that both expected and actual are NSArrays.
+        // - Convert the $in operator to the $eq operator.
+        if (![expected isKindOfClass:[NSArray class]]) {
+            expected = @[ expected ];
+        }
+        if (![actual isKindOfClass:[NSArray class]]) {
+            actual = @[ actual ? : [NSNull null] ];
```

Comment:
I changed it to `actual = actual ? @[ actual ] : @[ [NSNull null] ]` since the `@[]` causes an issue with the `$exists` operator functioning correctly.  Done in b667c92d0cf9994ca7a8751af6e7efedcb6f3199.